web/src/lib/utils/analytics.js

```diff
@@ -686,6 +686,8 @@ export default {
       click: {
         show_tax_calendar_modal: () =>
           sendEvent("tax_calendar_banner_button", "click", "show_tax_calendar_modal"),
+        show_myNJ_registration_prompt_modal: () =>
+          sendEvent("tax_calendar_banner_button", "click", "how_myNJ_registration_prompt_modal"),
```

Comment:
type here - this is important, the event won't send as EJ expects if this is wrong